CHANGES.rst

```diff
@@ -6,6 +6,11 @@ associations
 
 - Update diagrams to change sloper to detector1. [#4986]
 
+pipeline
```

Comment:
Minor nit, but `pipeline` entries should be further down in the log to maintain alphabetical order (i.e. after `datamodels`)

Reply:
D'oh!